### PR TITLE
cluster-1.yml#slack attribute

### PR DESCRIPTION
**Pull request by chassing**
* Add a dedicated `cluster-1` slack user group attribute to support clusters without GitHub authentication.
* Remove `slack-cluster-usergroups`

Tickets:
* slack-cluster-usergroups OIDC support ([APPSRE-6575](https://issues.redhat.com/browse/APPSRE-6575))
* Merge slack integrations ([APPSRE-6593](https://issues.redhat.com/browse/APPSRE-6593))

### CODE REVIEW

**Comment by maorfr** [comment]
would it make sense to make this implicit?
the reason we manage these user groups is to notify of upgrades. upgrades depend on `ocm`. so if a cluster has an `ocm` section, (implicitly) manage the user group `<cluster_name>-cluster`?

in addition, perhaps we could allow disabling of `slack-usergroups` in a cluster file (`disable.integrations`)?

tl;dr in my view, this PR may not be required, if you accept this proposal.

**Comment by chassing** [comment]
An implicit slack user group name is possible, and I can create a q-r method for that and reuse it in several integrations. 👍  But what about the channel? Currently, it comes from https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/dependencies/slack/coreos.yml#L62, which will be gone.

**Comment by maorfr** [comment]
the channel is just a place holder because (i think) there must be at least one channel

**Comment by chassing** [comment]
Yes, there must be one. Ok, then I'll reuse the `slack-workspace` config pattern.
